tests/rbd/test_configs/feature_enable_cli_test.yaml

```diff
@@ -0,0 +1,22 @@
+---
+name: rbd-feature-enable-test
+description: 'create pool and enable features'
+steps:
+  - command: ceph osd pool create
```

Comment:
I don't see the purpose of having seperate modules for commands if we specify command explicitly in test-config.
@psathyan what are your thoughts?

Reply:
@VasishtaShastry I agree with @HaruChebrolu review that this conflicts with the upcoming changes.